support-frontend/assets/components/checkoutNudge/checkoutNudgeContainer.tsx

```diff
@@ -92,27 +98,44 @@ export function CheckoutNudgeContainer({
 
 	let title, subtitle, paragraph;
 
-	if (dynamic) {
+	const calcWeeklyAmount = (annualAmount: number, toNearest?: number) => {
+		const weeklyAmount = Math.ceil((annualAmount * 100) / 52);
+
+		// Current/control solution is to round to the nearest 10p so this may be
+		// disposable if the variant is adopted as we want the dynamic version to be
+		// rounded to the nearest 1p
+		const roundedWeeklyAmount =
+			toNearest !== undefined
```

Comment:
I think it would be better to give `toNearest` a default value- presumably 1?- rather than to check for `undefined`.